packages/dd-trace/test/plugins/externals.json

```diff
@@ -266,7 +266,7 @@
   "mocha": [
     {
       "name": "mocha",
-      "versions": [">=5.2.0"]
+      "versions": [">=5.2.0", ">=8.0.0"]
```

Comment:
ℹ️ not picked up automatically by `install_plugin_modules` because I've spread the instrumentation of mocha in multiple files. I could probably fix `install_plugin_modules.js` but this is just simpler

Reply:
This is actually being addressed in a separate effort to separate tests and test setup from the versions in the implementation.